alacritty_terminal/src/term/mod.rs

```diff
@@ -1870,6 +1870,20 @@ impl ansi::Handler for Term {
         self.color_modified[index] = true;
     }
 
+    /// Get the foreground/background color
+    #[inline]
+    fn request_special_color<W: io::Write>(&mut self, writer: &mut W, code: u8, index: usize) {
```

Comment:
I'm not a huge fan of the method name here, I think `request_` indicates that the terminal is requesting something, however it's responding to a request.

Since they're often referred to as `dynamic` colors, I also think that `dynamic` is a better way to name this than `special`.

I think something like `dynamic_color_request` would be a better way to name this, however I'm open for suggestions here.

Reply:
Good point, I think `dynamic_color_sequence` is more indicative of what the function actually does as oposed to why it is being called. I've changed it to that, let me know if I need to change it again.